src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,13 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
```

Comment:
Ensure that your Bulldozer class has private fields with getters and setters if needed. Direct access to class fields should be avoided.